### ISSUE TITLE
Add scalafmt support

### ISSUE THREAD

**Issue by bzz** [src-d/gemini#50]
After #49, it would be nice to have auto-formatter

TODO
 - [ ] add/configure [`scalafmt`](http://scalameta.org/scalafmt) support
 - [ ] document IDE configuration http://scalameta.org/scalafmt/#IntelliJ as default InteliJ one can produce a bit different output
